### PR TITLE
WIP: Petition Letter Appendix

### PR DESCRIPTION
**Pull request by laooola**

### CODE REVIEW

File: app/node_modules/petition/template.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  displayName: 'Template',
+
+  propTypes: {
+    url: React.PropTypes.string,
+    context: React.PropTypes.object
+  },
+
+  getInitialState() {
+    return {
+      // error: null,
```
Comment (optionset) [inline app/node_modules/petition/template.js]: remove

File: app/node_modules/petition/template.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  render() {
+    const { template } = this.state;
+
+    // if(error) {
+    //   return (
+    //     <div className="container-fluid">
+    //       <Alert message="Fehler!" />
+    //     </div>
+    //   );
+    // }
```
Comment (optionset) [inline app/node_modules/petition/template.js]: move to `else` further down

File: app/node_modules/helpers/phone.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export const anonymizeTelephone = (phone, visibleCharacters, replaceWith) => {
```
Comment (optionset) [inline app/node_modules/helpers/phone.js]: - `anonymize`
- `phone` → `text`

File: app/node_modules/helpers/phone.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+  if (typeof(phone) === 'string') {
+    return phone.substring(0, visibleCharacters) + phone.substring(visibleCharacters).replace(/[0-9]/g, replaceWith);
```
Comment (optionset) [inline app/node_modules/helpers/phone.js]:
```js
const visiblePart = text.substring(0, visibleCharacters)
const anonymizedPart = text.substring(visibleCharacters).replace(/* ... */)
return visiblePart + anonymizedPart
```

File: app/node_modules/petition/appendix.js [hunk 5]
```diff
@@ -1,52 +1,64 @@
...
+        petition: {},
+        supporters: []
+      }
     };
   },
 
-  componentDidMount() {
-    const petitionId = _.get(this.props, 'params.id');
+  componentWillMount() {
+    this.getPetition(_.get(this.props, 'params.id'));
```
Comment (optionset) [inline app/node_modules/petition/appendix.js]: use `this.props.params.id`

**Comment by laooola** [comment]
ack?

**Comment by optionset** [comment]
do we have the real templates yet? and all data needed for the template?

**Pull request merged**